doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -273,13 +277,12 @@ There are multiple ways to generate and register device certificates:
    .. tab:: nRF91: Script generated private key
 
       .. warning::
-         When generating the private key on your computer, make sure to keep it secure and not to share it with anyone.
-         If the private key is compromised, the security of the device is compromised.
```

Comment:
Please keep the sentence about compromised keys. I've asked security about their input, including this warning.

Reply:
Not really a warning, but maybe "important".